trade-finance-manager-ui/templates/case/activity/activity-comment.njk

```diff
@@ -0,0 +1,39 @@
+{% from "govuk/components/textarea/macro.njk" import govukTextarea %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "index.njk" %}
+
+
+{% block sub_content %}
+<form
+    method="post"
+    data-cy="form">
+    {{ govukTextarea({
+    name: "comment",
+    id: "comment",
+    rows: "8",
+    label: {
+        text: "Add a comment",
+        classes: "govuk-label--l",
+        isPageHeading: false
+    },
+    hint: {
+        text: ""
+    },
+    attributes: {
+            'data-cy': 'activity-comment'
+        }
+    }) }}
```

Comment:
Looks like need to add validation handling annd probably max comment length?

Reply:
done